nixos/system-path: Drop default Perl package (#254942)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2311.section.md

- `writeTextFile` now requires `executable` to be boolean, values like `null` or `""` will now fail to evaluate.

> - The default system path does not contain `perl` anymore. Users who still need the package have to add it to `environment.systemPackages` explicitly.

Reviewer: ```suggestion
- The default system path does not contain `perl` anymore. Users who need the package in their path can add it to `environment.systemPackages` explicitly.
```

> - The default system path does not contain `perl` anymore. Users who still need the package have to add it to `environment.systemPackages` explicitly.

Reviewer: ```suggestion
- `perl` has been removed from the default system path. Users can add it to `environment.systemPackages` explicitly.
```

.. and this addition should prob be in `rl-2411.section.md`
